src/pages/Detail/Detail.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+// import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+import DetailTop from './components/DetailTop';
+import DetailPhoto from './components/DetailPhoto';
+import DetailMainLeft from './components/DetailMainLeft';
+import DetailMainRight from './components/DetailMainRight';
+import DetailReview from './components/DetailReview';
+import DetailMap from './components/DetailMap';
+import DetailNotice from './components/DetailNotice';
+import ModalWindow from '../../components/Modal/Modal';
+
+const Detail = () => {
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const [item, setItem] = useState([]);
+
+  // const params = useParams();
+
+  useEffect(() => {
+    fetch(`/data/detailMockData2.json`)
+      .then(response => response.json())
+      .then(data => setItem(data[0]));
+  }, []);
+
+  const {
+    name,
+    address,
+    detail_address,
+    price,
+    description,
+    latitude,
+    longitude,
+    maximum_occupancy,
+    bedroom,
+    bathroom,
+    bed,
+    host,
+    category,
+    room_type,
+    detail_images,
+    facilities,
+  } = item;
+
+  return item.length === 0 ? (
```

Comment:
```suggestion
  return item.length  ? (
```
이렇게 수정해도 똑같이 구현이 됩니다!

Reply:
저는 이걸 제거 해도 안되던데  내일 오전에 찾아 뵙겠습니다.

Reply:
```suggestion
  if (room_type) {
  return (
```
그 방법이 안된다면 위와 같은 방법을 사용해보세요
결국 값이 들어오지 않으면 화면을 보여주지 않는 로직이기 때문에
이와 같이 구현해도 될 것 같습니다